File: admin/types/strain.go
package types

type Strain struct {
	ID          string   `json:"id"`
	ProducerID  string   `json:"producer_id"`
	Name        string   `json:"name"`
	Category    string   `json:"category"`
	Genetics    string   `json:"genetics"`
	THC         float64  `json:"thc"`
	Terpenes    float64  `json:"terpenes"`
	Price       float64  `json:"price"`
	HarvestDate string   `json:"harvest_date"`
	Images      []string `json:"images"`
}

File: admin/app/src/types/strain.ts
export class Strain {
    id: string = '';
    producer_id : string = '';
    name: string = '';
    category: string = '';
    genetics: string = '';
    thc: number = 0;
    terpenes: number = 0;
    total_cannabinoids: number = 0;
    cbd: number = 0;
    terpene_list: string[] = new Array<string>(3);
    harvest_date: string = '';
    price: number = 0;
    images: string[] = [];
}